codec-http2/src/main/java/io/netty/handler/codec/http2/CompressorHttp2ConnectionEncoder.java

```diff
@@ -56,10 +61,16 @@ public CompressorHttp2ConnectionEncoder(Http2ConnectionEncoder delegate) {
 
     public CompressorHttp2ConnectionEncoder(Http2ConnectionEncoder delegate, int compressionLevel, int windowBits,
                                             int memLevel) {
+        this(delegate, compressionLevel, windowBits, memLevel, new Encoder.Parameters().setQuality(4));
+    }
+
+    public CompressorHttp2ConnectionEncoder(Http2ConnectionEncoder delegate, int compressionLevel, int windowBits,
+                                            int memLevel, Encoder.Parameters parameters) {
```

Comment:
Do you think it makes sense to accept a list of `CompressionOptions` instead?

Reply:
Ah, I forgot this part.